Tests/RabbitMq/ConsumerTest.php

```diff
@@ -341,4 +355,121 @@ public function testGracefulMaxExecutionWontWaitIfPastTheTimeout()
 
         $consumer->consume(1);
     }
+
+    public function testTimeoutWait()
```

Comment:
Please rerun this test and check the failing reason:
`1) OldSound\RabbitMqBundle\Tests\RabbitMq\ConsumerTest::testTimeoutWait
Expectation failed for method name is "wait" when invoked 2 time(s)
Parameter 2 for invocation PhpAmqpLib\Channel\AbstractChannel::wait(null, false, 29) does not match expected value.
Failed asserting that 29 matches expected 30.
/home/travis/build/eMAGTechLabs/RabbitMqBundle/RabbitMq/Consumer.php:95
/home/travis/build/eMAGTechLabs/RabbitMqBundle/Tests/RabbitMq/ConsumerTest.php:398`

Reply:
Check also:
`3) OldSound\RabbitMqBundle\Tests\RabbitMq\DynamicConsumerTest::testTimeoutWait
Expectation failed for method name is "wait" when invoked 2 time(s)
Parameter 2 for invocation PhpAmqpLib\Channel\AbstractChannel::wait(null, false, 29) does not match expected value.
Failed asserting that 29 matches expected 30.
/home/travis/build/eMAGTechLabs/RabbitMqBundle/RabbitMq/Consumer.php:95
/home/travis/build/eMAGTechLabs/RabbitMqBundle/Tests/RabbitMq/ConsumerTest.php:398`